manifests/agent.pp

```diff
@@ -275,11 +276,22 @@
     }
   }
 
-  if $facts['kernel'] == 'windows' and $manage_choco {
-    package { $zabbix_package_agent:
-      ensure   => $zabbix_version,
-      provider => $zabbix_package_provider,
-      tag      => 'zabbix',
+  if $facts['kernel'] == 'windows' {
+    if $manage_choco {
+      package { $zabbix_package_agent:
+        ensure   => $zabbix_version,
+        provider => $zabbix_package_provider,
+        tag      => 'zabbix',
+      }
+    } else {
+      package { $zabbix_package_agent:
+        ensure          => $zabbix_package_state,
+        require         => Class['zabbix::repo'],
```

Comment:
Why do we require zabbix::repo class here?

Reply:
We don't, removed.